### ISSUE TITLE
added_prompt and negative_prompt 

### ISSUE THREAD

**Issue by 1508189800** [yangxy/PASD#36]
Do we only need to add added_prompt and negative_prompt during inference, not during training?

**Comment by yangxy** [comment]
Yes, you don't need them during training. They are optional in inference (I would recommend to use them for better results).
